src/components/Header/Header.jsx

```diff
@@ -1,17 +1,36 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import Button from '../../common/Button/Button';
-import { LOGOUT } from '../../constants';
+import PropTypes from 'prop-types';
+import { LOGOUT, TOKEN_KEY, USER_NAME } from '../../constants';
 import Logo from './components/Logo/Logo';
 import styles from './styles.module.scss';
 
-const Header = () => (
-	<section className={styles.wrapperHeader}>
-		<Logo />
-		<div className={styles.wrapperButton}>
-			<span className={styles.nameAccount}>Julia</span>
-			<Button buttonText={LOGOUT} />
-		</div>
-	</section>
-);
+const Header = ({ setUserName, userName }) => {
+	let navigate = useNavigate();
+
+	const logout = () => {
+		localStorage.removeItem(TOKEN_KEY);
+		localStorage.removeItem(USER_NAME);
+		setUserName('');
+		navigate('/login');
+	};
+	return (
+		<section className={styles.wrapperHeader}>
+			<Logo />
+			<div className={styles.wrapperButton}>
+				<span className={styles.nameAccount}>{userName}</span>
+				{userName && userName !== '' && (
```

Comment:
if userName is truthy, then it cannot be "" (empty string), remove userName !== ''